src/dispatch/incident/messaging.py

```diff
@@ -63,28 +61,41 @@ def send_welcome_ephemeral_message_to_participant(
 
     # we send the ephemeral message
     convo_plugin = plugins.get(INCIDENT_PLUGIN_CONVERSATION_SLUG)
+
+    message_kwargs = {
+        "name": incident.name,
+        "title": incident.title,
+        "status": incident.status,
+        "type": incident.incident_type.name,
+        "type_description": incident.incident_type.description,
+        "priority": incident.incident_priority.name,
+        "priority_description": incident.incident_priority.description,
+        "commander_fullname": incident.commander.name,
+        "commander_weblink": incident.commander.weblink,
+        "document_weblink": incident.incident_document.weblink,
+        "storage_weblink": incident.storage.weblink,
+        "ticket_weblink": incident.ticket.weblink,
+        "conference_weblink": incident.conference.weblink,
+        "conference_challenge": incident.conference.conference_challenge,
+    }
+
+    faq_doc = document_service.get_incident_faq_document(db_session=db_session)
+    if faq_doc:
+        message_kwargs.update({"faq_weblink": faq_doc.weblink})
+
+    reference_doc = document_service.get_conversation_reference_document(db_session=db_session)
```

Comment:
In some functions you're using `faq_doc` and `reference_doc` and in others `incident_faq` and `conversation_reference `. Can you use the same everywhere?